lib/devices/cpu_device_manager.js

```diff
@@ -51,12 +51,15 @@ export default class CpuDeviceManager {
 
 class CpuDevice {
   constructor(id, active=false) {
-    this.id = id
+    this.id = id;
```

Comment:
I assumed you added the string conversion because it created serious issues when `id` wasn't a string... And this is easily missed in the future.